### ISSUE TITLE
Ways to do Conditional Checks within a component, or trigger a Value with a certain threshold. 

### ISSUE THREAD

**Issue by TehTurk** [Neos-Metaverse/NeosPublic#3024]
### Is your feature request related to a problem? Please describe.

Currently doing sound interactions with some objects on Neos require a logix script per object your interacting. Usually.you have to say the parameters based on a rotation threshold, or distance, or value etc.

### Describe the solution you'd like

Currently there no way to do any sort of conditional checks easily within components, it'd be nice if there was a way to do things like ==, >, <, >=, <= or != To a value and have it go true. Currently there is an == for Reference or Value, but it'd be ideal if there was a full option for folks to use. 

### Describe alternatives you've considered

There is ways to do it in Logix, but it's also sometimes faster/more simpler to have some sort of conditional check you can apply.

**Comment by Frooxius** [comment]
Are you looking for a component that would drive a value/reference depending on whether a reference value is bigger or smaller than the other one? Essentially a variant of ValueEqualityDriver for other comparison operations?

**Comment by TehTurk** [comment]
Yes! But also ones that can have >= <= and != As I expect them to be asked for in the future as well by others.
